### DISCUSSION TITLE
Reusing Description field on CSV Importer

### DISCUSSION THREAD

**Discussion by zcallen1** [firefly-iii/firefly-iii#5040] [Firefly III]
In some of my transactions, the Description field has enough information to key off of and map it to an opposing account. If I select "Description" as the role and try to map it, no fields show up.

If I select "Opposing Account (name)" as the role and then check the map option, the fields show up and I can now select accounts to map to.

However, upon importing I will lose the description. Is there a way to get both? I want the description to be populated by the description field, but also map some of the entries to an opposing account.

**Comment by JC5** [comment]
I'm afraid not, you would have to duplicate the column before you use it in the CSV importer.

**Reply by zcallen1** [reply]
Ok that was my first guess, but I wanted to confirm this before I started making duplicate columns.
